.github/workflows/pr.yml

```diff
@@ -103,4 +103,6 @@ jobs:
       - name: TestDotNetAliasesRenameComponent
         run: dotnet run integration test TestDotNetAliasesRenameComponent
       - name: TestDotNetAliasesRetypeParents
-        run: dotnet run integration test TestDotNetAliasesRetypeParents
\ No newline at end of file
+        run: dotnet run integration test TestDotNetAliasesRetypeParents
+      - name: TestDeletedWith
```

Comment:
Oh can we add "TestProvider" here as well, I missed that in the last change.

cc @Zaid-Ajaj is there a reason these are explicitly listed out rather than just running every integration test?

Reply:
@Frassle just for better visibility on each integration that gets shown in the github action. I think we can swap these out for `dotnet run all-integration-tests`